# Hypothesis testing Assignment
1) A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units. A randomly selected sample of cutlets was collected from both units and measured? Analyze the data and draw inferences at 5% significance level. Please state the assumptions and tests that you carried out to check validity of the assumptions.

Minitab File : **Cutlets.mtw**


In [3]:
import pandas as pd
from scipy import stats

In [4]:
data=pd.read_csv("/content/drive/MyDrive/Colab Notebooks/Cutlets (1).csv")

In [5]:
data.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [6]:
data1=data["Unit A"]

In [7]:
data2=data["Unit B"]

In [8]:
stats.ttest_ind(data1,data2)

Ttest_indResult(statistic=0.7228688704678061, pvalue=0.4722394724599501)

Here P value is 0.4722. So the P value is greater than the significance level, so it is failed to
reject null hypothesis. And we concluded that there is no significant difference in the diameter
of the cutlet between two units

# 2) A hospital wants to dettermine whether there is any difference in the average Turn Around time(TAT) of reports of the laboratories on their preffered list. They collected a random sample and recorded TAT for reports of 4 laboratories. TAT is defined as sample collected to report dispatch.


Analyze the data and determine whether there is any difference in average TAT among the difference laboratories at 5% significance level.

Minitab : LabTAT.mtw

In [9]:
lab=pd.read_csv("/content/drive/MyDrive/Colab Notebooks/LabTAT (1).csv")

In [10]:
lab.head()

Laboratory 1  Laboratory 2  Laboratory 3  Laboratory 4
0        185.35        165.53        176.70        166.13
1        170.49        185.91        198.45        160.79
2        192.77        194.92        201.23        185.18
3        177.33        183.00        199.61        176.42
4        193.41        169.57        204.63        152.60

In [11]:
stats.f_oneway(lab.iloc[:,0],lab.iloc[:,1],lab.iloc[:,2],lab.iloc[:,3])

F_onewayResult(statistic=118.70421654401437, pvalue=2.1156708949992414e-57)

P value=2.1156700949992414e-57, which is less than the significance level. So we have
to reject the null hypothesis. So we concluded that there is difference in average TAT
among the different laboratories.

3)Sales of products in four different regions is tabulated for males and females.
Find if male-female buyer rations are similar across regions.


In [13]:
data=pd.read_csv("/content/drive/MyDrive/Colab Notebooks/BuyerRatio.csv")

ANS: null hypothesis(h0):Data are normal alternate hypothesis(ha):data are not normal if p-value is > 0.05 => Accept null hypothesis if p-value is < 0.05 =>Reject null hypothesis alternate hypothesis is used when we take action or p-value is less than 0.05 As p-value is 0.227 > 0.05 P high Ho fly => Accept Ho,hence accept null hypothesis(H0) means data are normal 5)Normality test for West

As p-value is 0.227 > 0.05 P high Ho fly => Accept Ho,hence accept null hypothesis(H0) means data are normal 6)Normality test for North

As p-value is 0.227 > 0.05 P high Ho fly => Accept Ho,hence accept null hypothesis(H0) means data are normal

4)TeleCall uses 4 centers around the globe to process customer
order forms. They audit a certain % of the customer order
forms. Any error in order form renders it defective and has to
be reworked before processing. The manager wants to check
whether the defective % varies by centre. Please analyze the
data at 5% significance level and help the manager draw
appropriate inferences
Minitab File: CustomerOrderForm.mtw


In [12]:
data=pd.read_csv("/content/drive/MyDrive/Colab Notebooks/Costomer+OrderForm.csv")

ANS: Assume Null Hypothesis as Ho: Independence of categorical variables (customer order forms defective % does not varies by centre) Thus, Alternative hypothesis as Ha Dependence of categorical variables (customer order forms defective % varies by centre)

Inference: As (p_value = 0.2771) > (α = 0.05); Accept Null Hypthesis i.e. Independence of categorical variables Thus, customer order forms defective % does not varies by centre.

Fantaloons Sales managers commented that % of males versus females walking in to the store differ based on day of the week. Analyze the data and determine whether there is evidence at 5 % significance level to support this hypothesis. Minitab File: Fantaloons.mtw

ANS: 2-Proprotion Test H0:Proportion of male vs female in weekdays = Proportion of male vs female in weekends Ha:Proportion of male vs female in weekdays NOT = Proportion of male vs female in weekends

P-value is 0.968 > 0.05=>P high Ho fly => Accept Ho Hence Proportion of male vs female in weekdays = Proportion of male vs female in weekends